app/models/order_item.rb

```diff
@@ -1,4 +1,14 @@
 class OrderItem < ApplicationRecord
 	belongs_to :order
 	belongs_to :item
+
+	def self.top_three_items
```

Comment:
I think 3 is a perfect number for scale, and for not overwhelming our customers!